fbchat/_client.py

```diff
@@ -1715,6 +1715,23 @@ def sendLocalImage(
             thread_type=thread_type,
         )
 
+    def forwardAttachment(self, attachment_id, thread_id=None):
+        """
+        Forwards an attachment
+
+        :param attachment_id: Attachment ID to forward
+        :param thread_id: User/Group ID to send to. See :ref:`intro_threads`
+        :raises: FBchatException if request failed
+        """
+        thread_id, thread_type = self._getThread(thread_id, None)
+        data = {
+            "attachment_id": attachment_id,
+            "recipient_map[{}]".format(generateOfflineThreadingID()): thread_id,
+        }
+        j = self._post(
```

Comment:
Nothing useful in the response, I take?

Reply:
Only user IDs, but we don't need it

Reply:
Just checked, there's at least a `success` / `error` parameter.

Successful:
```
{
    "__ar": 1,
    "payload": {
        "success": true,
        "error": null,
        "user_ids": [
            "[REDACTED]"
        ]
    },
    "bootloadable": {},
    "ixData": {},
    "bxData": {},
    "gkxData": {},
    "qexData": {},
    "lid": "[REDACTED]"
}
```

Failed:
```
{
    "__ar": 1,
    "payload": {
        "success": false,
        "error": "Fanout recipient cannot be the same as the mailbox id",
        "message_targets": [
            "[REDACTED]"
        ]
    },
    "bootloadable": {},
    "ixData": {},
    "bxData": {},
    "gkxData": {},
    "qexData": {},
    "lid": "[REDACTED]"
}
```

So maybe we should extend `check_json` to account for this format as well?